geonode_mapstore_client/client/js/routes/Home.jsx

```diff
@@ -422,6 +435,19 @@ function Home({
             <div className="gn-main-home">
 
                 <div className="gn-container">
+                    <div className="gn-row">
```

Comment:
We should apply a body-bg color to this component so could add a new class called gn-home-section to this div

```html
<div className="gn-row gn-home-section">
```

This is the expected mixin to apply in the geonode.scss file
```scss
@include background-color-var(map.get($theme-vars, 'body-bg'), --gn-body-bg);
```